examples/basecoin/app/app.go

```diff
@@ -23,44 +23,50 @@ const (
 	appName = "BasecoinApp"
 )
 
+var (
+	cdc *wire.Codec
+)
+
+func init() {
+	cdc = MakeTxCodec()
+}
+
 // Extended ABCI application
 type BasecoinApp struct {
 	*bam.BaseApp
-	cdc *wire.Codec
 
 	// keys to access the substores
 	capKeyMainStore *sdk.KVStoreKey
-	capKeyIBCStore  *sdk.KVStoreKey
```

Comment:
better to comment out and add a note about using it in the future instead of just removing